### ISSUE TITLE
Negative value of max-visible

### ISSUE THREAD

**Issue by bobsaintcool** [emersion/mako#130]
Hi!

Starting playing with Mako I'm not sure if I'm geeting all of the wording beind the `max-visible` things:

> If -1, all notifications are visible

When I set this to `-X` and then execute something like `notify-send "foo"` mako display the notification and an extra "(X+1 more)" as shown in the following screenshot:

![Untitled](https://user-images.githubusercontent.com/13803031/55290331-73e9b380-53c1-11e9-9efe-ddb124ee21cd.png)

Not sur if its unwanted or I didn't get the meaning of this option.

Thanks for the contribution with Mako!

**Comment by vilhalmer** [comment]
`-1` is the only negative value that works on purpose, and means that you will never see (N more). Any other value should be positive, e.g. `max-visible=3` means that it will show up to three notifications and then past that they will be hidden behind the (N more) notification.

We should make other negative values fail to parse.

**Comment by bobsaintcool** [comment]
Thanks,

At this time, with mako 1.2 I see the same behavior when `max-visible=-1`:

```~/.config/mako/config
max-visible=-1 
```

Then:

```
$ mako
```

And:

```
$ makoctl dismiss -a 
$ notify-send 'foo' 
```

And get the following result, show the (N more) as follow:

![Untitled](https://user-images.githubusercontent.com/13803031/55290490-6f25ff00-53c3-11e9-8c39-c155744215c7.png)

Maybe 2 bugs so?

**Comment by vilhalmer** [comment]
Yep, I can reproduce this behavior in 1.2. It seems to have gotten fixed at some point, so 1.3 will behave correctly.
